packages/test/end-to-end-tests/src/test/batching.spec.ts

```diff
@@ -61,13 +65,13 @@ const tests = (args: ICompatLocalTestObjectProvider) => {
 
     beforeEach(async () => {
         // Create a Container for the first client.
-        const container1 = await args.makeTestContainer(registry);
+        const container1 = await args.makeTestContainer(testContainerConfig);
```

Comment:
Yeah... I think I can solve his particular problem by providing the old OpProcessingController  as well from the LocalTestObjectProvider.

Reply:
Long term, I would like to keep the type to be a mix of new and old so that we can detect back-compat API problems.
Test code will have to be modified if the API changes any way.  That means that we will need to do what the user needs to do change their code using feature detection.